### ISSUE TITLE
Support for multi-unit setups

### ISSUE THREAD

**Issue by gravelfreeman** [normen/obs-mcu#7]
For example I'm using the X-Touch with an X-Touch Extender that gives me 8 more channels.

In Logic I can add them seamlessly. I believe there's a way to do it in OBS Studio/obs-mcu as well, but it might require a code update.

It would be ideal to have the capacity to use everything that's on my desk when I'm running a production with OBS!

**Comment by normen** [comment]
Not possible right now but I'll add that to the list. How does your setup look that you have so many audio channels at once?

**Comment by normen** [comment]
Really, the audio mixing in obs isn't fantastic and it can't be with all those unsynced sources. I'd always prefer mixing at least speakers and such externally, even with something like ardour and its jack I/O if you want it all ITB. I have different setups from a multi-unit apollo frontend that sends the mix via S/PDIF to OBS to a version with an X32 Rack as the frontend directly connected via USB to the OBS machine. That gives you 4 virtual devices that at least are in sync by default.

Still, has nothing to do with the fact that supporting extender setups would be nice. 😄

**Comment by gravelfreeman** [comment]
> Really, the audio mixing in obs isn't fantastic and it can't be with all those unsynced sources.

Yeah that's really a bummer. If I can afford it and upgrade my XR18 with the upcoming Wing rack I could get more i/o and bypass completely the OBS mixer. Send busX via USB and add that single audio source in OBS.

> I have different setups from a multi-unit apollo frontend that sends the mix via S/PDIF to OBS to a version with an X32 Rack as the frontend directly connected via USB to the OBS machine. That gives you 4 virtual devices that at least are in sync by default.

Looks like a great setup you have! I'm wondering how it would work in my situation where I've got external callers. The only way I've found so far is to buy a multi-output interface and wire each output to individual inputs on the XR18 and have each callers go through my mixer. Then send the final mix in a bus/usb to OBS. The only way to achieve this is by using the OBS plugin audio-monitor with which you can send each caller to different audio output devices. It's a lot of overhead to bypass the OBS mixer in my situation but it might be worth a try in the future.

In the meantime, thanks to obs-mcu solving mostly everything!
